packages/modules/pt/src/pages/citizen/Create/Response.js

```diff
@@ -0,0 +1,162 @@
+import React, { useEffect } from "react";
+import { Card, Banner, CardText, SubmitBar, LinkButton } from "@egovernments/digit-ui-react-components";
+import { Link } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import { Loader } from "@egovernments/digit-ui-react-components";
+//import getPDFData from "../../../getPDFData";
+
+const GetActionMessage = () => {
+  const { t } = useTranslation();
+  return t("CS_FILE_DESLUDGING_APPLICATION_SUCCESS");
+};
+
+const BannerPicker = (props) => {
+  return (
+    <Banner
+      message={GetActionMessage()}
+      applicationNumber={props.data?.Properties[0].propertyId}
+      info={props.t("CS_FILE_DESLUDGING_APPLICATION_NO")}
+      successful={props.isSuccess}
+    />
+  );
+};
+
+const Response = ({ data, onSuccess }) => {
+  const { t } = useTranslation();
+  const tenantId = Digit.ULBService.getCurrentTenantId();
+  let mutation = {};
+
+  useEffect(() => {
+    try {
+      const { subtype, pitDetail, address, pitType, source, ownershipCategory, owners } = data;
+
+      const loc = address?.locality.code;
+      const formdata = {
+        Property: {
+          tenantId: address?.city.code,
+          address: {
+            city: address?.city.name,
+            doorNo: address?.doorNo,
+            buildingName: "NA",
+            locality: {
+              code:
+                address?.city.code === "pb.amritsar"
```

Comment:
@Tulika-eGov  didn't you receive code of the selected mohalla , don't use substring method